tests/test_enterprise/api/test_views.py

```diff
@@ -3458,6 +3458,9 @@ def test_bulk_enrollment_in_bulk_courses_pending_licenses(
             self.assertEqual(expected_response, response_json)
         self.assertEqual(len(PendingEnrollment.objects.all()), expected_num_pending_licenses)
 
+        if expected_num_pending_licenses == 1:
```

Comment:
Please add similar logic to check the other enrollment source you changed in the test of the view that isn't bulk enrollment.